src/lib/src/common/workspace.spec.ts

```diff
@@ -76,3 +108,52 @@ describe('Workspace.getSubpaths()', () => {
   });
 
 });
+
+
+
+describe('Workspace.getParent()', () => {
+
+      it('returns the parent element', () => {
```

Comment:
is there a tests that makes sure that getParent does not go any 'higher' than the workspace root (which could be a non empty path?)